arcgis-ios-sdk-samples/Geometry/Nearest vertex/README.md

```diff
@@ -23,8 +23,17 @@ Tap anywhere on the map. An orange cross will show at that location. A blue circ
 ## Relevant API
 
 * AGSGeometry
-* AGSGeometryEngine
 * AGSProximityResult
+* class AGSGeometryEngine.geodeticDistanceBetweenPoint1(_:point2:distanceUnit:azimuthUnit:curveType:)
+* class AGSGeometryEngine.nearestCoordinate(in:to:)
+* class AGSGeometryEngine.nearestVertex(in:to:)
+* class AGSGeometryEngine.normalizeCentralMeridian(of:)
+
+## Additional information
+
+The `distance` property in `AGSProximityResult` is planar (Euclidean) distance. Planar distance only works for geometries that have a defined projected coordinate system, which preserves distance and length. The geometries in this demo are defined in Web Mercator, which doesn't preserve distance. Hence we need to use the geodetic distance APIs to get the accurate distances.
```

Comment:
```suggestion
The value of `AGSProximityResult.distance` is planar (Euclidean) distance. Planar distance only works for geometries that have a defined projected coordinate system, which preserves distance and length. The geometries in this demo are defined in Web Mercator, which doesn't preserve distance. Hence we need to use the geodetic distance APIs to get the accurate distances.
```

Also, the information about the type of distance isn't in the doc for that property. Could you please create an issue to add it?

Reply:
Do you mean to create an issue in the cocoa repo? 

I saw there is an open issue `runtime/bistromath/issues/1132` that aims at improving the doc for this API, but I'm not sure how it works. Will this doc change be made in the core and got picked up by the audit, or do we directly change it in cocoa?

Reply:
These kinds of questions would be better asked off of GitHub.

Reply:
Sorry, removed! 😶‍🌫️